contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
+
+Critical to making this happen is to deliver each tool without compromises. The way to achieve this is very simple - shipping every product that teams use, from day one when there are just two technical cofounders. This is because if we get in with these developers, we’re upstream of every need they’ll have. The end result is that we can offer all the products in one place, with no need to charge as much as everyone else for each one, and most importantly with perfect quality customer data. An actual perfect 360 customer view.
+
+Even so, it’s right to question whether this actually does any good for the world. Are we really in need of better software? And beyond those in the most competitive industries, will perfect customer data actually make a difference to an everyday person’s ability to build a successful business?
+
+No and not really. However, that misses the point, unless you understand the secret master plan alluded to above. The strategy of PostHog is to get early stage software teams to use every single customer related product inside our platform, where we can easily generate a perfect customer data record. This means we can learn to configure and automate how all of these tools work. We can simplify the experience of running a business.
+
+Then with each successive product, we can reach a wider and wider market and further simplify the experience of running a business until anyone can do it.
+
+I’d like to address a repeated argument against shipping multiple products, let alone all of them.
+
+### You can build more than a product or two
+
+The world of tech is changing fast.
+
+First, there’s just more money at better valuations than when this advice was created. Software companies, us included, have more leverage than ever before - with capital deployed well, we can do more than our predecessors.
+
+Second, software development is far quicker.
+
+Third, we’ve managed this very well so far. 
 
 ## Our strategy
 
-### 1. Provide every tool needed to build successful products
+### 1. Be the source of truth for customer and product data
+
+Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete customer record. This has happened because the entire industry has focused on integration instead of consolidation.
+
+Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
```

Comment:
Is "hundreds" an exaggeration for the bottom 99% of businesses? If so we should probably remove.

Reply:
```suggestion
Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data they collect, simply because they are misused and divorced from the source of truth. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
```

I feel like we need an explanation of why product analytics is still relevant (it is!!!)

Reply:
disagree on hundreds, we'll wind up upmarket where this is the case - that part will eventually matter to us